packages/standalone-link/docs.mdx

```diff
@@ -0,0 +1,21 @@
+---
+componentName: 'StandaloneLink'
+---
+
+As navigational element directly following content.
+
+<LiveComponent>{`<StandaloneLink href="https://hashicorp.com">Call-to-action</StandaloneLink>`}</LiveComponent>
+
+<UsageDetails packageJson={packageJson} />
+
+<PropsTable props={componentProps} />
+
+## Rules
+
+- They may appear on their own, directly following content, or in combination with a primary Button
+
+## UX writing guidelines
+
+- Target a maximum of 2-3 words.
+- Target a readability grade level score of 8 to meet plain language guidelines.
+- Text uses Sentence case capitalization.
```

Comment:
🙌 Oooh these are good to include for not only us, but for guiding external parties as well (which is the main benefit, I think)